packages/components/src/components/text-area/_text-area.scss

```diff
@@ -99,6 +99,12 @@
       color: transparent;
     }
   }
+
+  .#{$prefix}--text-area__label-wrapper {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+  }
```

Comment:
One last thing! Can you just duplicate these styles over to the `packages/styles`? That will be the V11 style package, and I want to ensure this feature is available in V11.

Reply:
@tw15egan correct me if I am wrong but I thought I did?

https://github.com/carbon-design-system/carbon/pull/10442/files#r791899919

Reply:
is the the correct path/file?